### ISSUE TITLE
VyOS refusing to route traffic despite following guide to the letter

### ISSUE THREAD

**Issue by Reinitialized** [reinitialized-studio/reinitialized.net#3]
I have followed two [different](https://docs.vyos.io/en/latest/quick-start.html) [guides](https://blog.kroy.io/2020/05/04/vyos-from-scratch-edition-1/) in attempts to quickly deploy a temporary network while work is being done to automate reinitialized.net. VyOS router is successfully able to ping and access WAN resources, but devices on LAN are unable to reach outside networks. Need to investigate whether this is a routing issue or a firewall issue.

**Comment by Reinitialized** [comment]
Upon further investigation of the [kroy.io guide](https://blog.kroy.io/2020/05/04/vyos-from-scratch-edition-1/), the following potential discrepancies should be investigated:
- validate VyOS is able to access LAN devices (LOCAL-LAN firewall group)
- validate NAT source is configured to the right parameters
- validate physical connections are labelled correctly

Upon further review when writing instructions for this issue, I've noticed 1.4-rolling has seemingly disappeared from the VyOS downloads. If the above does not solve this issue, download [this specific release](https://downloads.vyos.io/rolling/current/amd64/vyos-1.3-rolling-202101052023-amd64.iso).

**Comment by Reinitialized** [comment]
A precise problem was never determined. I have written it off as a typo, as I reinstalled VyOS to the most recent rolling release at the time and followed the guide to the letter.

 Another potential problem area is the release I was working with could not be found on VyOS downloads, nor could a few other 1.4 releases. It's possible some broken releases were released and ran into the issue during my deployment.
